must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -133,6 +133,14 @@ for ns in $namespaces; do
                 pids_ceph+=($!)
                 { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd snap ls --all $image --pool $bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-snap-"${image}"-debug.log 2>&1 &
                 pids_ceph+=($!)
+                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror image status $bp/$image --format=json" >> "${JSON_COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-image-status-"${bp}"-"${image}"-debug.log 2>&1 &
+                pids_ceph+=($!)
+                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror snapshot schedule list --pool=$bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-snap-schedule-list-"${bp}"-debug.log 2>&1 &
```

Comment:
`rbd mirror snapshot schedule list` , `rbd mirror pool status` and `rbd mirror pool info $bp` is per pool not per image, This need to be done in for loop of the pool not on for loop of image.

Reply:
I have moved these commands one level up, under the pool for loop, this works but I don't have those files created since I don't have those resources, it would be great if you can test this on a working cluster that has all the relevant resources.